src/redux/reducers/ConceptReducers.js

```diff
@@ -24,7 +24,7 @@ import {
   EDIT_CONCEPT_CREATE_NEW_NAMES,
   EDIT_CONCEPT_REMOVE_ONE_NAME,
   REMOVE_CONCEPT,
-  ADD_NEW_ANSWER_MAPPING,
+  // ADD_NEW_ANSWER_MAPPING,
```

Comment:
@AGMETEOR , kindly remove the commented code.

Reply:
@Duncanian I have removed it.